pkgdb2/api/extras.py

```diff
@@ -585,3 +586,29 @@ def api_monitored():
             '\n'.join(output),
             content_type="text/plain;charset=UTF-8"
         )
+
+
+@API.route('/dead/package/<pkg_name>/<clt_name>')
+def api_dead_package(pkg_name, clt_name):
+    '''
+    Returned the content of the of dead.package file
+    -----------------------
+    Retired packages should have in their git a ``dead.package`` file
+    containing the explanation as why the package was retired.
+    This method calls cgit to return that explanation.
+
+    ::
+
+        /api/dead/package/acheck/master
+
+    '''
+    req = requests.get(
```

Comment:
A thought:  would it make more sense to return a ``302`` redirect here instead getting the content ourselves?